Assets/Scripts/Model/Timing/AccessCard.cs

```diff
@@ -51,7 +51,7 @@ async private Task Trash()
 
         async private Task Steal()
         {
-            if (card is cards.corp.CorporateSalesTeam)
+            if (card.Type.Stealable != Stealable.CANNOT_STEAL)
```

Comment:
I'm thinking, that instead of just being an `enum`,  the `Stealable` could be an actual `interface` with various implementations. E.g. `Bellona` would give the runner a choice to pay the additional credits cost and `City Works Project` would deal the meat damage. The default "just steal" option would also be one of the implementations. It'd actually incorporate the code below:
```c#
card.FlipFaceUp();
game.runner.zones.score.Add(card, 2);
```

And "cannot steal me" would also be one of the impls.

Reply:
I am not sure I have understood...
Are you ok if I fix first the above problem and then try to fix this?

Reply:
Sure!